js/main.js

```diff
@@ -26,4 +32,75 @@ function saveEntry(event) {
   $img.classList.replace('object-contain', 'object-cover');
   data.nextEntryId++;
   $form.reset();
+  $ul.prepend(renderEntry(entryObj));
+  viewSwap('entries');
+  toggleNoEntries(1);
+}
+
+function renderEntry(entry) {
+  const $li = document.createElement('li');
+  const $div1 = document.createElement('div');
+  const $div2 = document.createElement('div');
+  const $div3 = document.createElement('div');
+  const $img = document.createElement('img');
+  const $h2 = document.createElement('h2');
+  const $p = document.createElement('p');
+
+  $div1.setAttribute('class', 'row');
+  $div2.setAttribute('class', 'column-half');
+  $div3.setAttribute('class', 'column-half');
+  $img.setAttribute('src', entry.imageUrl);
+  $img.setAttribute('alt', entry.title);
+  $h2.textContent = entry.title;
+  $p.textContent = entry.notes;
+
+  $li.append($div1);
+  $div1.append($div2, $div3);
+  $div2.append($img);
+  $div3.append($h2, $p);
+
+  return $li;
+}
+
+document.addEventListener('DOMContentLoaded', renderAllEntries);
+
+function renderAllEntries(event) {
+
+  const entriesCount = data.entries.length;
+
+  toggleNoEntries(entriesCount);
+
+  for (let i = 0; i < entriesCount; i++) {
+    $ul.append(renderEntry(data.entries[i]));
+  }
+
+  viewSwap(data.view);
+}
+
+function toggleNoEntries(entriesCount) {
+  if (entriesCount === 0) {
+    $noEntries.classList.remove('hidden');
+  } else {
+    $noEntries.classList.add('hidden');
+  }
+}
```

Comment:
`toggleNoEntries` should not take any arguments when called and should not have any parameters. You should use your `data.entries` array to determine whether or not you have any entries